Gemfile

```diff
@@ -41,18 +41,18 @@ group :documentation do
 end
 
 group :system_tests do
-  gem 'beaker', '~> 4.x',             :require => false
```

Comment:
This would only become a problem if/when beaker 5 comes out, the 3 to 4 transition wasn't painful but required changes to testing.

Reply:
We'll find that during the automated tests and can always pin if needed.